### DISCUSSION TITLE
Getting all purchases of a user in a specific shop via OAuth

### DISCUSSION THREAD

**Discussion by minisbett** [etsy/open-api#1138] [Q&A]
Hey!

I'm currently looking into implementing a temporary solution for an upcoming EU law that might not be implemented by Etsy properly. For that, I have a website set up where the user can log into using Etsy. I need to display all ongoing purchases of a user in a specific shop for that. Which API endpoint would I need to use? I saw that there's the `billing_r` scope, which is probably what I need, but I cannot find any endpoint.

Kind regards

**Comment by Gareth064** [comment]
When you say an Etsy "user", do you mean a Buyer or a Seller?

When it comes to "Orders", the API is only there to serve the seller and their individual shop. The OAuth process is to give your app access to a Sellers shop, not give it access to a normal logged in Etsy Buyer account.

**Reply by minisbett** [reply]
hm. What options do I have? I'm referring to a Buyer.
